composer.json

```diff
@@ -19,15 +19,15 @@
         }
     ],
     "require": {
-        "php": ">=5.6.4",
+        "php": ">=7.2.0",
         "guzzlehttp/guzzle": "^6.2",
-        "illuminate/notifications": "5.3.*|5.4.*",
-        "illuminate/queue": "5.1.*|5.2.*|5.3.*|5.4.*",
-        "illuminate/support": "5.1.*|5.2.*|5.3.*|5.4.*"
+        "illuminate/notifications": ">5.5.0|^6.0",
```

Comment:
`>5.5.0` is not valid because it will fit 7.0 too.

Reply:
yeah, you're right. I changed it